easybuild/easyconfigs/m/matplotlib/matplotlib-3.9.2-gfbf-2024a.eb

```diff
@@ -18,6 +18,7 @@ builddependencies = [
 
 dependencies = [
     ('Python', '3.12.3'),
+    ('pybind11', '2.12.0'),
```

Comment:
is it really not a builddep?

Reply:
Unless `pip check` trips over it, let's make it a build-only dep